python/source/org/libreoffice/ide/eclipse/python/PythonProjectHandler.java

```diff
@@ -0,0 +1,387 @@
+ * License version 2.1, as published by the Free Software Foundation.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+ * MA 02111-1307 USA
+ *
+ * The Initial Developer of the Original Code is: Sun Microsystems, Inc..
+ *
+ * Copyright: 2002 by Sun Microsystems, Inc.
+ *
+ * All Rights Reserved.
+ *
+ * Contributor(s): Cedric Bosdonnat
+ *
+ *
+ ************************************************************************/
+package org.libreoffice.ide.eclipse.python;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Vector;
+
+import javax.xml.xpath.XPathConstants;
+import javax.xml.xpath.XPathException;
+import javax.xml.xpath.XPathExpression;
+import javax.xml.xpath.XPathFactory;
+
+import org.eclipse.core.resources.IFile;
+import org.eclipse.core.resources.IFolder;
+import org.eclipse.core.resources.IProject;
+import org.eclipse.core.resources.IProjectDescription;
+import org.eclipse.core.resources.IWorkspaceRoot;
+import org.eclipse.core.resources.ResourcesPlugin;
+import org.eclipse.core.runtime.CoreException;
+import org.eclipse.core.runtime.IPath;
+import org.eclipse.core.runtime.IProgressMonitor;
+import org.eclipse.core.runtime.Path;
+import org.eclipse.jdt.core.IClasspathEntry;
+import org.eclipse.jdt.core.IJavaProject;
+import org.eclipse.jdt.core.JavaCore;
+import org.eclipse.jdt.core.JavaModelException;
+import org.eclipse.jdt.launching.JavaRuntime;
+import org.libreoffice.ide.eclipse.core.PluginLogger;
+import org.libreoffice.ide.eclipse.core.model.IUnoFactoryConstants;
+import org.libreoffice.ide.eclipse.core.model.IUnoidlProject;
+import org.libreoffice.ide.eclipse.core.model.ProjectsManager;
+import org.libreoffice.ide.eclipse.core.model.UnoFactoryData;
+import org.libreoffice.ide.eclipse.core.model.config.IOOo;
+import org.libreoffice.ide.eclipse.core.model.language.IProjectHandler;
+import org.libreoffice.ide.eclipse.python.build.OOoContainerPage;
+import org.libreoffice.ide.eclipse.python.registration.RegistrationHelper;
+import org.libreoffice.ide.eclipse.python.tests.TestsHelper;
+import org.w3c.dom.Node;
+import org.xml.sax.InputSource;
+
+/**
+ * The Project handler implementation for Java.
+ */
+public class PythonProjectHandler implements IProjectHandler {
+
+    private static final String P_REGISTRATION_CLASSNAME = "regclassname"; //$NON-NLS-1$
+    private static final String P_JAVA_VERSION = "javaversion"; //$NON-NLS-1$
+
+    private static final String[] KEPT_JARS = { "unoil.jar", //$NON-NLS-1$
```

Comment:
The array of Jar file names has been removed and also the function isKeptJar(String pJarName) where it was being used has been removed. Since Jar files are required only for Java Project we will not have any use of it in Python Project